ArduPlane/altitude.cpp

```diff
@@ -907,3 +937,24 @@ float Plane::get_landing_height(bool &rangefinder_active)
 
     return height;
 }
+
+/*
+  if a terrain location doesn't have the relative_alt flag set
+  then fix the alt and trigger a flow of control error
+ */
+void Plane::fix_terrain_WP(Location &loc, uint32_t linenum)
```

Comment:
I wonder if this should be on the location object? It can fetch home itself.

Reply:
I don't want to pollute the non-Plane code with this special case logic for plane